pure_pursuit_wrapper/src/pure_pursuit_wrapper_node.cpp

```diff
@@ -28,14 +28,26 @@ int main(int argc, char** argv) {
   ros::NodeHandle nh("");
 
   pure_pursuit_wrapper::PurePursuitWrapper PurePursuitWrapper(nh);
+  ros::CARMANodeHandle::setSpinRate(10);
```

Comment:
Line 28, should that be updated to CARMANodeHandle instead of NodeHandle, or does it not matter?

Reply:
@maefromm Good catch. That is a change that should be done.